query-engine/connector-test-kit/src/test/scala/writes/nestedMutations/SubqueryTooManyColumnsSpec.scala

```diff
@@ -0,0 +1,227 @@
+package writes.nestedMutations
```

Comment:
If you think these tests are necessary and can't be merged into existing ones:
- Put them in a new package , e.g. `writes/regressions/<issuenumber>Spec` with a comment on top briefly explaining what is the issue.
- Convert all the tests to `legacy = false` versions. I will reject all tests going forward that are written with legacy queries.

Reply:
Done.